project_report/tex/design.tex

```diff
@@ -1 +1,16 @@
 \section{Design}
+
+\subsection{Dashboard design}
+A very important aspect the group has had to concider, is which design pattern
+to model the dashboard after. A design pattern is a form of template, which in
+our case, dictates how the different classes communicate. A relevant design
+pattern is the MVC (Model - View - Controller). This pattern separates the
+database (model), the logic (controller) and the GUI (View) into different
```

Comment:
Model = data-related logic
View = UI logic
Controller = interface between the above

A more detailed explanation can be found [here](https://www.tutorialspoint.com/mvc_framework/mvc_framework_introduction.htm)